alacritty/src/ipc.rs

```diff
@@ -40,13 +45,39 @@ pub fn spawn_ipc_socket(options: &Options, event_proxy: EventLoopProxy<Event>) -
 
     // Spawn a thread to listen on the IPC socket.
     thread::spawn_named("socket listener", move || {
-        // Accept up to 2 bytes to ensure only one byte is received.
-        // This ensures forward-compatibility.
-        let mut buf = [0; 2];
-
-        while let Ok(received) = socket.recv(&mut buf) {
-            if buf[..received] == SOCKET_MESSAGE_CREATE_WINDOW {
-                let _ = event_proxy.send_event(Event::new(EventType::CreateWindow, None));
+        let mut data = Vec::new();
+        for stream in listener.incoming() {
+            let stream = match stream {
+                Ok(stream) => stream,
+                Err(err) => {
+                    warn!("Failed to get incomming stream! {}", err);
+                    continue;
+                },
+            };
```

Comment:
When does this error? I'd assume when the listener is killed or something?

I feel like we can either just `.filter` on `incoming()` or the error is actually an issue and we `break`?

Reply:
we continue, just log a warning that some client failed to connect to listener, which is our main application.

Reply:
You know of any example this can be reproduced? Is this generally just an issue with the client and safe to skip as a server?